Source/monster.cpp

```diff
@@ -497,7 +497,7 @@ void PlaceUniqueMonst(int uniqindex, int miniontype, int bosspacksize)
 	if (uniqueData.mlevel != 0) {
 		monster.mLevel = 2 * uniqueData.mlevel;
 	} else {
-		monster.mLevel += 5;
+		monster.mLevel = monster.MData->mLevel + 5;

```

Comment:
Why remove stuff when you can fix it?

Reply:
because duplicate code is bad...

Reply:
what duplicate code xD

Reply:
Wasn't this value already set correctly elsewhere?

Reply:
? it adds +5 to original lvl + difficulty bonus

Reply:
Ok, cool then lets remove one so that it only happens oncs.

Reply:
then you will have an issue with `monster.mLevel = 2 * uniqueData.mlevel;`

Reply:
I feel like you are leaving out some details in your explanation. As I understand it the default code looks like this:

```cpp
monster.mLevel = monster.MData->mLevel + 5;
monster.mLevel += 5;
if (isUnique) {
    PlaceUniqueMonst(monster.uniqindex, monster.miniontype, monster.bosspacksize) {
	if (uniqueData.mlevel != 0) {
		monster.mLevel = 2 * uniqueData.mlevel;
	} else {
		monster.mLevel += 5;
	}
    }
}
```

If that is the case removing the second `monster.mLevel = monster.MData->mLevel + 5;` should be a nice clean fix with no side effects. If it is not the case then please explain what is actually going on, so that I can understand why this is a proper fix.

Reply:
> Then you obviously lose that +5 lvl for quest monsters and difficulty bonus is still applied twice, so this is as far from a fix as possible xD

Obviously not if you have the assumptions I do, it's pointless to tell me that it's sill with out explaining to me what the reality is. If you would prefer that I look it up my self you will have to wait till I have time to do so.

> Default code looks like this

Thank you